doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -580,6 +589,18 @@ To launch C application next steps should be performed.
 
 <div id="run-java" class="tab-pane fade" markdown="1" >
 
+To launch Java application next steps should be performed.
+
+1. Save application code into FirstKaaDemo.java file into `demo_app` directory.
+1. Navigate to `demo_app` directory.
+1. Build application.
+
+        javac -cp *.jar *.java
+
+1. Launch application.
+        
+        java -cp '.:./*' FirstKaaDemo
```

Comment:
Please note that in Unix and Windows systems different path and file separator chars are used. For path: ':' and ';', for file '/' and '\' accordingly.